File: dust/sample/app/controllers/Application.java
package controllers;

import org.codehaus.jackson.node.ObjectNode;

import play.*;
import play.mvc.*;
import play.libs.*;
import views.html.*;

public class Application extends Controller {

	public static Result frame() {
		return ok(index.render());
	}

	public static Result data() {
		ObjectNode json = Json.newObject();

		json.put("name", "Json");
		json.put("count", 1);

		return ok(json);
	}

}

File: dust/sample/app/controllers/Application.scala
package controllers

import play.api._
import play.api.mvc._

object Application extends Controller {

  def index = Action {
    Ok(views.html.index())
  }

  def data = Action {
    import play.api.libs.json._
    Ok(
      Json.obj(
        "name" -> "Json",
        "count" -> 1
      )
    )
  }

}